### PR TITLE
ABDK Audit: Lock issues

### PR DESCRIPTION
**Pull request by hensha256**
Issue 16 only

### CODE REVIEW

File: contracts/base/Dispatcher.sol [hunk 1]
```diff
@@ -55,7 +55,7 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks,
                             payerIsUser := calldataload(add(inputs.offset, 0x80))
                         }
                         bytes calldata path = inputs.toBytes(3);
-                        address payer = payerIsUser ? lockedBy : address(this);
+                        address payer = payerIsUser ? lockedBy() : address(this);
```
Comment (ewilz) [inline contracts/base/Dispatcher.sol:58]: I'm confused about this fix. Isn't it impossible to get here without it being locked? It's an internal function that's only called by a function that locks the contract.

Comment (ewilz) [reply contracts/base/Dispatcher.sol:58]: (also issue 5 seems to be about Payments, not locks)

Comment (hensha256) [reply contracts/base/Dispatcher.sol:58]: issue 5 is _row_ 6 because of the header row. The issue numbers are in column E :)

**Review by snreynolds — APPROVED** [review]
lgtm

**Pull request merged**